docs/source/guide/mmm/resources.md

```diff
@@ -37,18 +37,22 @@ To add more substance to our session and ensure it's not just informative but al
 
 [![Youtube video thumbnail image](https://img.youtube.com/vi/ikCK76gq65Q/maxresdefault.jpg)](https://youtu.be/ikCK76gq65Q?si=fitQK_GrQcoSNWJq)
 
+## Business Cases
+
+* PyMC Labs: [Bayesian Media Mix Models: Modelling changes in marketing effectiveness over time](https://www.pymc-labs.io/blog-posts/modelling-changes-marketing-effectiveness-over-time/)
+* Carlos Eduardo Trujillo Agostini: [Better budgeting with Bayesian models: Bolt’s story with PyMC-Marketing](https://bolt.eu/en/blog/budgeting-with-bayesian-models-pymc-marketing/)
```

Comment:
I would replace name with company, i.e. Bolt:
Shell & DataBricks:

Reply:
Changed in https://github.com/pymc-labs/pymc-marketing/pull/562/commits/c14472ebcdbe34a9242b8095c2db0532e4a43bcd